src/migration/1721903257825-Vordr.ts

```diff
@@ -0,0 +1,44 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Vordr1721903257825 implements MigrationInterface {
+  name = 'Vordr1721903257825'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "user" ADD "flags" jsonb NOT NULL DEFAULT '{"trustScore": 1, "vordr": false}'`);
```

Comment:
we usually avoid having default be anything then `{}` because then if you are inserting row with some other flags data eg. `{ bla: 1 }` then it will overwrite default. So either make it `{}` or we have to be careful in all other places when inserting User to not overwrite the flags and always send these defaults as well.

Reply:
we could leave it to just `{"trustScore": 1}`, since `vordr: false` and `vord: undefined` is both falsy.

Reply:
I can leave it to empty, but in user create set `{"trustScore": 1}`, and just do a query to insert for the rest? In future it would be Vordr that calculates the score on the fly.

Reply:
Changed default to be `{}`, and instead on user creation I insert the default values

Reply:
Maybe it will filter everyones comments because falsy?

Reply:
If `trustScore` is undefined, this `user.flags?.trustScore <= 0` will return `false`, and not filtered.

Reply:
Oh, default flag on comment. I'll double check.